scripts/install_used_gem_versions

```diff
@@ -10,7 +10,7 @@ source_filenames = (Dir.glob(File.join(__dir__,"../books/**/*")) + [File.join(__
                        .select{|filename| !File.directory?(filename)}
 
 gem_name_to_versions_array_hash = source_filenames.each_with_object({}) do |filename,hash|
-  source = File.read(filename)
+  source = File.read(filename, :encoding => 'utf-8')
```

Comment:
Can you make it the `encoding: 'utf-8'` style?  More modern.  Need to hook up rubocop for this repo.